### ISSUE TITLE
Segfault on large account

### ISSUE THREAD

**Issue by bdowne01** [rebuy-de/aws-nuke#40]
When running aws-nuke against a large account with many resources, aws-nuke dies during the resource enumeration (specifically Lambda functions) with this error:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xc4ca02]

goroutine 37 [running]:
github.com/rebuy-de/aws-nuke/resources.(*RDSNuke).ListInstances(0xc4202021e0, 0xc420054b40, 0xc4215bdfb0, 0xfacb01, 0xca7ed2, 0xc420189101)
	/go/src/github.com/rebuy-de/aws-nuke/resources/rds-instances.go:26 +0x152
github.com/rebuy-de/aws-nuke/resources.(*RDSNuke).ListInstances-fm(0xd759e0, 0xc420054b40, 0xc4215bdfb0, 0xe, 0x0)
	/go/src/github.com/rebuy-de/aws-nuke/resources/listers.go:95 +0x2a
github.com/rebuy-de/aws-nuke/cmd.Scan.func1(0xc420116180, 0xc4204003b0, 0xc420054b40)
	/go/src/github.com/rebuy-de/aws-nuke/cmd/scan.go:22 +0x77
created by github.com/rebuy-de/aws-nuke/cmd.Scan
	/go/src/github.com/rebuy-de/aws-nuke/cmd/scan.go:39 +0x9e

**Comment by svenwltr** [comment]
Hello @bdowne01 

It looks like this is an issue with the RDS instance. Or to be more precisely the reading of `DBName`.

* Could you please run this command with the AWS CLI and see if there is any entry which is empty or even null? `aws rds describe-db-instances --query 'DBInstances[*].DBName'`
* Does aws-nuke die every time with that account?
* Could you try to nuke lambda only by specifying the `--target` flag? ie `aws-nuke --target LambdaNuke ...`

**Comment by bdowne01** [comment]
Happy to help.

- I think I see what some of the issue, based on the RDS query you provided.   RDS SQL Server instances do not contain a "DBName" key.  The result is nothing returned at the CLI, even though there are a dozen SQL Server instances.

- It does die each time, in the same spot.

- Running `aws-nuke with --target LambdaFunction` again dies at the same spot, but with a slightly different error:

```
./aws-nuke -c config.yaml --profile sandbox --target LambdaFunction
... snip ...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xc4ca02]
goroutine 44 [running]:
github.com/rebuy-de/aws-nuke/resources.(*RDSNuke).ListInstances(0xc4202021e0, 0xc420184780, 0xc4202effb0, 0xfacb01, 0xca7ed2, 0xc42038bb01)
	/go/src/github.com/rebuy-de/aws-nuke/resources/rds-instances.go:26 +0x152
github.com/rebuy-de/aws-nuke/resources.(*RDSNuke).ListInstances-fm(0xd759e0, 0xc420184780, 0xc4202effb0, 0xe, 0x0)
	/go/src/github.com/rebuy-de/aws-nuke/resources/listers.go:95 +0x2a
github.com/rebuy-de/aws-nuke/cmd.Scan.func1(0xc42012a180, 0xc4203fc2d0, 0xc420184780)
	/go/src/github.com/rebuy-de/aws-nuke/cmd/scan.go:22 +0x77
created by github.com/rebuy-de/aws-nuke/cmd.Scan
	/go/src/github.com/rebuy-de/aws-nuke/cmd/scan.go:39 +0x9e
```

**Comment by bdowne01** [comment]
This appears to be fixed in version 1.2.1.  Thanks!
